content/news/vcmi-0.100-release.md

```diff
@@ -0,0 +1,35 @@
++++
+categories = ["Releases", "Stable"]
+date = "2022-09-10"
+description = ""
+featured = "vcminewterrain.jpg"
+featuredalt = ""
+featuredpath = "date"
+linktitle = ""
+title = "VCMI 0.100 released"
+type = "post"
+forumurl = "https://forum.vcmi.eu/"
+
++++
+
+## We're happy to announce new release!
+
+Few years passed after we released VCMI 0.99 and now it's time for a new release.
+Tons of changes, improvements, and fixes have been introduces during that time, it's even hard to recall all of them.
+However we'd like to highlight several major changes.
+
+## Artificial intellengence
+You are welcome to try brand new AI algorithm called Nullkiller or you can play with classic VCAI computer player.
+Both algorithms became much smarter in comparison with last release, so just choose from the laucher settings whatever you like and enjoy.
```

Comment:
```suggestion
Both algorithms became much smarter in comparison with the last release, so just choose from the launcher settings whatever you like and enjoy.
```

Reply:
Something does not click: first, you write "NullKiller is a **brand** new algorithm", and now you write "Both algorithms got smarter in comparison to the last release". So, the question is "is NullKiller a new algorithm that appeared in 0.100? or it was included into 0.99?". In the former case, "Both got smarter..." is false, in the latter case "a brand new" is not true.

Reply:
became much smarter -> was significantly improved